src/index.ts

```diff
@@ -1,7 +1,20 @@
 import { Docs } from './docs.js';
-// TODO: Read design script from CLI
-import { CustomDesign } from './custom-design.js';
+import * as path from 'node:path';
+import { DesignContext, DocDesign } from './doc-design.js';
 
-(function() {
-  Docs.createDocs(new CustomDesign().getDesign());
+const getUsername = () => {
```

Comment:
Minor and it can be done later:
We can have a `design-context.ts`, and then we can add a tiny unit test for this, too.

Reply:
Good call!  Done.